package.json

```diff
@@ -112,6 +111,8 @@
     "webpack": "4.44.1"
   },
   "resolutions": {
-    "@types/react": "16.9.46"
+    "@types/react": "16.9.46",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
```

Comment:
Prefer fixed versioning that matches the dependency versions above. Renovate will manage this going forward.